src/components/metadata/styles/metadata.scss

```diff
@@ -0,0 +1,181 @@
+@import '../../../styles/variables';
+
+.pipeline-metadata {
+  position: absolute;
+  top: -1px; /* Avoids pixel rounding gaps */
+  right: 0;
+  bottom: -1px;
+  z-index: 1;
+  display: flex;
+  flex-direction: column;
+  width: $sidebar-width-open;
+  padding: 12px 0 24px 0;
+  overflow-y: auto;
+  transform: translateX(100%);
+  transition: transform ease 0.4s;
+
+  @media (min-width: 900px) {
+    &--visible {
+      transform: translateX(0);
+    }
+  }
```

Comment:
Yeah the responsive approach here needed some work. I've made some changes that affect both sidebars and the chart zoom approach, but I think it works well.

The best way is to try it out first, but here are roughly how it works (this sounds more complicated then it really is):

- the main sidebar
  - covers the screen at `viewport <= 400px`

- the metadata sidebar 
  - now has a close button
  - covers the screen at `viewport <= 400px`
  - begins to overlay main sidebar at `viewport <= 800px`

- the chart zoom
  - moves and zooms the chart to fit in the space between either sidebar at `viewport > 400px`
  - as above but excludes sidebars at `viewport <= 400px` (such that they just overlay)

The implementation of this is simple just using `min/max-width` on the sidebars and allowing them to flex naturally, rather than specific media queries.